mlir/lib/Dialect/Polynomial/IR/PolynomialAttributes.cpp

```diff
@@ -202,11 +202,30 @@ Attribute RingAttr::parse(AsmParser &parser, Type type) {
     polyAttr = attr;
   }
 
+  Polynomial poly = polyAttr.getPolynomial();
+  APInt root(coefficientModulusAttr.getValue().getBitWidth(), 0);
+  bool hasRoot = succeeded(parser.parseOptionalComma());
```

Comment:
Nit: looks like this can be folded into the `if`.

Reply:
Fixed in afedacc7478